controllers/articles-controllers.js

```diff
@@ -29,14 +31,33 @@ exports.getArticleCommentsById = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  const query = req.query
-  fetchArticles(query)
-    .then((articles) => {
-      res.status(200).send(articles);
-    })
-    .catch((err) => {
-      next(err);
+  const query = req.query;
+  const queryKeys = Object.keys(query);
+  if (queryKeys.length === 0) {
+    fetchArticles(query)
```

Comment:
Why would you pass query if its length is zero? In that case, there is nothing to pass